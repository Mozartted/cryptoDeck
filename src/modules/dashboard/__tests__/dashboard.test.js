import React from 'react';
import {shallow} from 'enzyme';
// import Button from './Button';
import Dashboard from "../views/layouts/dashboard"
describe('Dashboard Screens', () => {
    describe('Rendering', () => {
        it('should match to snapshot', () => {
            const props = {
                 tokens: {
                    data: [],
                    status: {},
                },
                loading: false,
            }
            const component = shallow(<Dashboard 
                tokens={props.tokens} 
                loading={props.loading}  
                getTokens={(start) => console.log(start)}
                />)
            expect(component).toMatchSnapshot()
        });
    });
});