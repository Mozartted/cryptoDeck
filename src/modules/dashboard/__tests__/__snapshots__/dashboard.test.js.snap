// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dashboard Screens Rendering should match to snapshot 1`] = `ShallowWrapper {}`;
